metron-platform/metron-parsing/metron-parsing-storm/src/test/java/org/apache/metron/writers/integration/SimpleHbaseEnrichmentWriterIntegrationTest.java

```diff
@@ -58,7 +58,7 @@
   /**
    * {
    *     "parserClassName": "org.apache.metron.parsers.csv.CSVParser",
-   *     "writerClassName": "org.apache.metron.enrichment.writer.SimpleHbaseEnrichmentWriter",
+   *     "writerClassName": "org.apache.metron.writer.hbase.SimpleHbaseEnrichmentWriter",
```

Comment:
I believe this would constitute a breaking change.  Need to add a note on this in the Upgrading doc.

Reply:
Done